### PR TITLE
Create roles.yml

### LINKED ISSUE #2141: add OBO ops roles to the website
see slack thread
https://obo-communitygroup.slack.com/archives/C01B6MQCDLN/p1664967782409519

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #2141

### CODE REVIEW

File: _data/roles.yml
Comment (cthoyt) [inline _data/roles.yml:18]: you could always make this better with more structured information, like a list of the current people who have the role

Comment (nicolevasilevsky) [reply _data/roles.yml:19]: I separated these into open opportunities and currently filled roles. Let me know what you think and please suggest any other changes, thanks!

File: _data/roles.yml
Comment (cthoyt) [inline _data/roles.yml:70]: use the fact that this is YAML to make this into a list instead of just text with commas (same for all of the other ones)

File: _data/roles.yml
Comment (cthoyt) [reply _data/roles.yml:19]: i'd say better to have the fact that the role is looking for new people to be a property of the role itself and not the way that roles are grouped together (so maybe add a new key `open: true` or `open: false`)

Comment (nicolevasilevsky) [reply _data/roles.yml:20]: I revised this

File: _data/roles.yml
Comment (nicolevasilevsky) [reply _data/roles.yml]: done

File: _data/roles.yml
Comment (cthoyt) [inline _data/roles.yml:67]: next step is to turn these lists of strings into lists of dictionaries so we can include extra information about each person

```yaml
lead(s):
 - name: Jie Zheng
   status: [lead / support]
 - name: Nico Matentzoglu
   status: [lead / support]
```

Comment (nicolevasilevsky) [reply _data/roles.yml]: cool. I added this where I knew the info.

@jamesaoverton @zhengj2007 @nlharris @matentzn could you please add the data for yourselves, where appropriate? thanks

**Comment by nicolevasilevsky** [comment]
@cthoyt is there a way to preview what this would like like on the website?

**Comment by cthoyt** [comment]
Right now this is just data and doesn't have an associated web page that formats it nicely. In a follow-up PR I will make this

File: _data/roles.yml
Comment (jamesaoverton) [reply _data/roles.yml]: I added my status and dates

File: _data/roles.yml
Comment (matentzn) [inline _data/roles.yml:15]: I would like to propose an element `responsibilities` - I will fill it in. It will detail what needs to be done for each roll exactly. So when yo assume a role you know exactly the 10 things that need to be done.

File: _data/roles.yml
Comment (matentzn) [inline _data/roles.yml:60]: gh handle

File: _data/roles.yml
Comment (cthoyt) [inline _data/roles.yml:109]: delete this line

File: _data/roles.yml
Comment (cthoyt) [inline _data/roles.yml:70]: ORCID and github plz

Comment (matentzn) [reply _data/roles.yml:70]: Please don't add any new people yet. There will be a public process with official invites etc. For now list me everywhere and I will manage the transition with an official process

File: _data/roles.yml
Comment (nicolevasilevsky) [reply _data/roles.yml:1]: done

File: _data/roles.yml
Comment (nicolevasilevsky) [reply _data/roles.yml:19]: I added a line for this to each role. Thanks for filling it in @matentzn

File: _data/roles.yml
Comment (nicolevasilevsky) [reply _data/roles.yml]: I revised this

(do I need the @?)

File: _data/roles.yml
Comment (nicolevasilevsky) [reply _data/roles.yml]: done

File: _data/roles.yml
Comment (nicolevasilevsky) [reply _data/roles.yml]: I added the github and orcids

@bvarner-ebi and @shawntanzk could you please add your orcid?

Comment (cthoyt) [reply _data/roles.yml]: I would say don't spend time on metadata collection past ORCID since we can automate grabbing any other relevant metadata, but here they are:

- @shawntanzk https://orcid.org/0000-0001-7258-9596
- @zhengj2007 https://orcid.org/0000-0002-2999-0103
- @bvarner-ebi https://orcid.org/0000-0002-1773-2692

**Comment by cthoyt** [comment]
@nicolevasilevsky if you update this branch from master, I can send another PR that improves the display and does some remaining curation

**Comment by matentzn** [comment]
updated.

**Comment by nicolevasilevsky** [comment]
> @nicolevasilevsky if you update this branch from master, I can send another PR that improves the display and does some remaining curation

done!

**Comment by cthoyt** [comment]
@nicolevasilevsky can we merge this as a good first step then make follow-up issues?

**Comment by nicolevasilevsky** [comment]
(I can't merge myself)

**Comment by cthoyt** [comment]
@nicolevasilevsky Nico will be gone for a week or two on personal time, so does this mean that all OBO Operations will now completely halt until he's back? Do we have an explicit list of people who we can get to merge this in the mean time? In addition, including that list as a separate, explicitly annotated "role" (i.e., someone with merge rights) would be valuable.

**Comment by nicolevasilevsky** [comment]
@cthoyt I hope we can move things forward in Nico's absence!

We can bring this up on the OBO ops call tomorrow. I'll add an agenda item.

I think @jamesaoverton and @nlharris have merging rights? Could one of you merge this PR? Thanks!

**Comment by nlharris** [comment]
I'm not authorized to merge. Clearly we need to identify people who are!

**Comment by jamesaoverton** [comment]
@nlharris I believe you are authorized to merge, because you are a member of the obo-foundry-operations-committee Team. Merging is currently blocked because nobody has reviewed and approved the PR: "Merging can be performed automatically with 1 approving review." I believe you also have permission to review and approve the PR.

My personal opinion is that we should discuss this one on the Operations call before merging.

**Comment by nlharris** [comment]
It says "You’re not [authorized](https://docs.github.com/articles/about-protected-branches/) to merge this pull request", which I thought meant I wasn't authorized to merge, but your explanation that it can't be merged because the PR hasn't been approved yet seems plausible.

Sorry I can't be on the OBO ops call today; I'm on another call.

**Comment by cthoyt** [comment]
@jamesaoverton @nlharris @nicolevasilevsky  maybe would be helpful if you could give me triage status on this repo. Right now, my approval doesn't technically mean anything.

**Review by cthoyt — APPROVED** [review]

**Comment by jamesaoverton** [comment]
@cthoyt You have Triage permissions according to the current settings.

**Review by jamesaoverton — APPROVED** [review]
We discussed this on the OBO Operations call, and agreed that breaking down these roles and getting clear about them is a very good idea. Storing the roles in YAML is also a good idea.

@matentzn, @nicolevasilevsky, @nlharris, and I have checked the current version, and I will merge it. Of course, we will revise and improve as we go.

**Comment by cthoyt** [comment]
@jamesaoverton sorry to bring this up again but I really don't understand how OBO Operations calls work. Why don't people on OBO Operations calls just engage in a public discussion on GitHub, attached to the code/data changes?

**Pull request merged**
